CRM/Aoservicelisting/Form/ProviderApplicationForm.php

```diff
@@ -85,10 +80,6 @@ public function setDefaultValues() {
         $defaults['work_address[1]'] = $primaryWorkAddress['street_address'];
         $defaults['postal_code[1]'] = $primaryWorkAddress['postal_code'];
         $defaults['city[1]'] = $primaryWorkAddress['city'];
-        $primaryWebsite = civicrm_api3('Website', 'get', ['contact_id' => $this->organizationId, 'url' => ['IS NOT NULL' => 1], 'sequential' => 1]);
```

Comment:
@Edzelopez this i think may need to go back as i think we still want org website stored as a website right?